assets/src/components/searchForm.tsx

```diff
@@ -215,8 +176,8 @@ const SearchFormFromStateDispatchContext = ({
         dispatch(setSearchText(""))
         onClear?.(event)
       }}
-      onFiltersChanged={(name, _) => {
-        dispatch(setSearchProperty(filterNameToSearchProperty(name)))
+      onFiltersChanged={(name, _isActive) => {
+        dispatch(setOldSearchProperty(name))
```

Comment:
Will update this to dispatch a different action to update the new property list as a different PR